### PR TITLE
204 add more words

### PR DESCRIPTION
**Pull request by omen273**

### CODE REVIEW

File: app/src/main/res/raw/data.json [hunk 1]
```diff
@@ -9699,23 +9702,1585 @@
...
+        }}
+    ],
+	[
+        {"EN":{
+            "word": "milligram",
+            "topics": ["maths and measurement", "science and technology"],
+            "level": "intermediate(B1)"
+        }},
+        {"RU": {
+            "word": "милиграмм",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:11128]:
```suggestion
            "word": "миллиграмм",
```

File: app/src/main/res/raw/data.json [hunk 2]
```diff
@@ -9699,23 +9702,1585 @@
...
+        }}
+    ],
+	[
+        {"EN":{
+            "word": "milliliter",
+            "topics": ["maths and measurement", "science and technology"],
+            "level": "intermediate(B1)"
+        }},
+        {"RU": {
+            "word": "милилитр",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:11139]:
```suggestion
            "word": "миллилитр",
```

File: app/src/main/res/raw/data.json [hunk 3]
```diff
@@ -9699,23 +9702,1585 @@
...
+            "topics": ["математика и измерения", "наука и техника"]
+        }}
+    ],
+	[
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:11154]:
```suggestion
    ],
	[
```

Comment (greenEkatherine) [reply app/src/main/res/raw/data.json:11154]: duplicated

File: app/src/main/res/raw/data.json [hunk 4]
```diff
@@ -9699,23 +9702,1585 @@
...
+        {"EN":{
+            "word": "millimeter",
+            "topics": ["maths and measurement", "science and technology"],
+            "level": "intermediate(B1)"
+        }},
+        {"RU": {
+            "word": "милиметр",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:11161]:
```suggestion
            "word": "миллиметр",
```

File: app/src/main/res/raw/data.json [hunk 5]
```diff
@@ -7620,31 +7620,34 @@
 	[
     {"EN":{
         "word": "thousand",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:7622]: I think, all former "numbers" are  good to add to "maths and measurement"

File: app/src/main/res/raw/data.json [hunk 6]
```diff
@@ -9699,23 +9702,1585 @@
...
+            "word": "percentage",
+            "topics": ["maths and measurement", "science and technology"],
+            "level": "intermediate(B1)"
+        }},
+        {"RU": {
+            "word": "процент",
+            "topics": ["математика и измерения", "наука и техника"]
+        }}
+    ],
+	[
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:11198]:
```suggestion
    ],
	[
```

Comment (greenEkatherine) [reply app/src/main/res/raw/data.json:11198]: duplicated

File: app/src/main/res/raw/data.json [hunk 7]
```diff
@@ -9699,23 +9702,1585 @@
...
+            "word": "gadget",
+            "topics": ["science and technology"],
+            "level": "intermediate(B1)"
+        }},
+        {"RU": {
+            "word": "гаджет",
+            "topics": ["наука и техника"]
+        }}
+    ],
+	[
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:10791]:
```suggestion
    ],
	[
```

Comment (greenEkatherine) [reply app/src/main/res/raw/data.json:10791]: duplicated, did you check it?

File: app/src/main/res/raw/data.json [hunk 8]
```diff
@@ -9699,23 +9702,1585 @@
...
+            "word": "medicine",
+            "topics": ["science and technology"],
+            "level": "elementary(A2)"
+        }},
+        {"RU": {
+            "word": "медицина",
+            "topics": ["наука и техника"]
+        }}
+    ],
+	[
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:10681]:
```suggestion
    ],
	[
```

File: app/src/main/res/raw/data.json [hunk 9]
```diff
@@ -9699,23 +9702,1585 @@
...
+            "word": "data",
+            "topics": ["computers", "science and technology"],
+            "level": "elementary(A2)"
+        }},
+        {"RU": {
+            "word": "данные",
+            "topics": ["электроника", "наука и техника"]
+        }}
+    ],
+	[
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:10406]:
```suggestion
    ],
	[
```

File: app/src/main/res/raw/data.json [hunk 10]
```diff
@@ -9699,23 +9702,1585 @@
...
+            "word": "kilogram",
+            "topics": ["maths and measurement", "science and technology"],
+            "level": "elementary(A2)"
+        }},
+        {"RU": {
+            "word": "килограмм",
+            "topics": ["математика и измерения", "наука и техника"]
+        }}
+    ],
+	[
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:10285]:
```suggestion
    ],
	[
```

File: app/src/main/res/raw/data.json [hunk 11]
```diff
@@ -9699,23 +9702,1585 @@
...
+            "word": "mile",
+            "topics": ["science and technology"],
+            "level": "beginner(A1)"
+        }},
+        {"RU": {
+            "word": "миля",
+            "topics": ["наука и техника"]
+        }}
+    ],
+	[
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:9900]:
```suggestion
    ],
	[
```

File: app/src/main/res/raw/data.json [hunk 12]
```diff
@@ -9523,12 +9526,12 @@
     [
         {"EN":{
             "word": "addition",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:9528]: please remove - duplicated and wrond level

File: app/src/main/res/raw/data.json [hunk 13]
```diff
@@ -9699,23 +9702,1585 @@
...
+            "level": "intermediate(B1)"
+        }},
+        {"RU": {
+            "word": "сложение",
+            "topics": ["математика и измерения", "наука и техника"]
+        }}
+    ],
+	[
+        {"EN":{
+            "word": "calorie",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:11046]: please union it with the same word in cooking topic

File: app/src/main/res/raw/data.json [hunk 14]
```diff
@@ -9699,23 +9702,1585 @@
...
+            "word": "virus",
+            "topics": ["science and technology"],
+            "level": "elementary(A2)"
+        }},
+        {"RU": {
+            "word": "вирус",
+            "topics": ["наука и техника"]
+        }}
+    ],
+	[
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:10021]:
```suggestion
    ],
	[
```

File: app/src/main/res/raw/data.json [hunk 15]
```diff
@@ -9699,23 +9702,1585 @@
...
+        }},
+        {"RU": {
+            "word": "компьютер",
+            "topics": ["наука и техника"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "menu",
+            "topics": ["science and technology"],
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:9760]: + "computers"?

File: app/src/main/res/raw/data.json [hunk 16]
```diff
@@ -9699,23 +9702,1585 @@
...
+            "level": "beginner(A1)"
+        }},
+        {"RU": {
+            "word": "меню",
+            "topics": ["наука и техника"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "mouse",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:9770]: + "computers"?

File: app/src/main/res/raw/data.json [hunk 17]
```diff
@@ -9699,23 +9702,1585 @@
...
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "open",
+            "topics": ["science and technology"],
+            "level": "beginner(A1)"
+        }},
+        {"RU": {
+            "word": "открыть файл",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:9786]: I think translation is not correct, it means "open a file", not just "open". I would remove this word

File: app/src/main/res/raw/data.json [hunk 18]
```diff
@@ -9699,23 +9702,1585 @@
...
+            "word": "add",
+            "topics": ["science and technology"],
+            "level": "beginner(A1)"
+        }},
+        {"RU": {
+            "word": "добавить",
+            "topics": ["наука и техника"]
+        }}
+    ],
+	[
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:9856]:
```suggestion
    ],
	[
```

File: app/src/main/res/raw/data.json [hunk 19]
```diff
@@ -9699,23 +9702,1585 @@
...
+        {"EN":{
+            "word": "times",
+            "topics": ["maths and measurement", "science and technology"],
+            "level": "intermediate(B1)"
+        }},
+        {"RU": {
+            "word": "умножить",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:11271]: раз или разы лучше

File: app/src/main/res/raw/data.json (hunk 5 above)
Comment (omen273) [reply app/src/main/res/raw/data.json:7622]: +1

File: app/src/main/res/raw/data.json [hunk 20]
```diff
@@ -7440,224 +7440,242 @@
 	[
     {"EN":{
         "word": "one",
-        "topics": ["numbers"]
+        "level": "beginner(A1)",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:7443]: Is level before topics ok?

File: app/src/main/res/raw/data.json [hunk 21]
```diff
@@ -9699,23 +9709,1464 @@
...
+            "word": "calorie",
+            "topics": ["maths and measurement", "science and technology"],
+            "level": "intermediate(B1)"
+        }},
+        {"RU": {
+            "word": "калория",
+            "topics": ["математика и измерения", "наука и техника"]
+        }}
+    ],
+	[
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:10963]:
```suggestion
    ],
	[
```

**Review by greenEkatherine — APPROVED** [review]

File: app/src/main/res/raw/data.json (hunk 20 above)
Comment (greenEkatherine) [reply app/src/main/res/raw/data.json:7443]: it's ok

**Pull request merged**